### ISSUE TITLE
Error creating a source

### ISSUE THREAD

**Issue by victordzmr** [stripe/stripe-mock#75]
When I create a source, the following error occurs. I'm using the [Docker image](https://hub.docker.com/r/stripemock/stripe-mock/) (latest tag, last updated 16 hours ago). The request body can be found in the trace.

```
Request: POST /v1/sources
Request data = map[owner:map[phone:+6383190290671 address:map[line1:911 Everardo Loaf Suite 890 postal_code:03302 country:CO] email:dayana80@example.com name:Rocio Schneider] token:tok_visa_debit type:card]
2018/05/28 10:27:56 http: panic serving 172.18.0.2:47992: reflect: call of reflect.Value.Type on zero Value
goroutine 2267 [running]:
net/http.(*conn).serve.func1(0xc4207960a0)
	/usr/local/go/src/net/http/server.go:1697 +0xd0
panic(0x7665c0, 0xc4201822c0)
	/usr/local/go/src/runtime/panic.go:491 +0x283
reflect.Value.Type(0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/reflect/value.go:1694 +0x1a3
github.com/stripe/stripe-mock/generator/datareplacer.isSameType(0x74ce40, 0xc42015c250, 0x0, 0x0, 0xc420879aa8)
	/go/src/github.com/stripe/stripe-mock/generator/datareplacer/datareplacer.go:49 +0xa9
github.com/stripe/stripe-mock/generator/datareplacer.ReplaceData(0xc420822210, 0xc420822a20, 0xc42065c3d8)
	/go/src/github.com/stripe/stripe-mock/generator/datareplacer/datareplacer.go:38 +0x147
github.com/stripe/stripe-mock/generator/datareplacer.ReplaceData(0xc4208221b0, 0xc420822990, 0x76b120)
	/go/src/github.com/stripe/stripe-mock/generator/datareplacer/datareplacer.go:25 +0x1d4
main.(*DataGenerator).Generate(0xc420a3fb18, 0xc420a3fc28, 0x0, 0x0, 0x0, 0x1)
	/go/src/github.com/stripe/stripe-mock/generator.go:122 +0x208
main.(*StubServer).HandleRequest(0xc42055af40, 0x993e40, 0xc4201f6000, 0xc420168000)
	/go/src/github.com/stripe/stripe-mock/server.go:201 +0x6ce
main.(*StubServer).HandleRequest-fm(0x993e40, 0xc4201f6000, 0xc420168000)
	/go/src/github.com/stripe/stripe-mock/main.go:75 +0x48
net/http.HandlerFunc.ServeHTTP(0xc4201a0ef0, 0x993e40, 0xc4201f6000, 0xc420168000)
	/usr/local/go/src/net/http/server.go:1918 +0x44
net/http.(*ServeMux).ServeHTTP(0x9e3fe0, 0x993e40, 0xc4201f6000, 0xc420168000)
	/usr/local/go/src/net/http/server.go:2254 +0x130
net/http.serverHandler.ServeHTTP(0xc420930d00, 0x993e40, 0xc4201f6000, 0xc420168000)
	/usr/local/go/src/net/http/server.go:2619 +0xb4
net/http.(*conn).serve(0xc4207960a0, 0x9944c0, 0xc42067ab80)
	/usr/local/go/src/net/http/server.go:1801 +0x71d
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2720 +0x288
```

By the way, I think it would be a good idea to tag releases in Docker Hub.

**Comment by brandur-stripe** [comment]
Thanks for reporting. Could you try the fixed that I shipped in #76 (v0.16.2)? Unfortunately, it's not clear to me how to reproduce this one from the test suite (documentation on Go's `reflect` package is _very_ light on how to produce a "zero Value"), but I think that'll do it.

> By the way, I think it would be a good idea to tag releases in Docker Hub.

Yeah, agreed. Unfortunately I haven't had the time to build out a more sophisticated build system.

**Comment by victordzmr** [comment]
It works fine now. Thank you very much! :)
